packages/backend/src/email/reset-password-mail.ts

```diff
@@ -0,0 +1,16 @@
+import { getEnv } from '@helpers';
+import { BaseMail } from './base-mail';
+
+export class ResetPasswordMail extends BaseMail {
+  protected setOptions() {
+    this._options = {
+      subject: `Reset Password | ${getEnv('APP_NAME')}`,
+    };
+  }
+
+  setText(text: string): this {
+    const resetMessage = `Link for resetting password: ${text}`;
```

Comment:
please use localisation. Refer to the new localisation rules

Reply:
Discussed about our localization with Vitalli and decided to remake it in more effective, cleaner way.